wordpress-mobile/WordPress-iOS
# Site creation: domain suggestion screen

Fixes #8181

Step 4 in the new site creation flow.

To test:
- In the My Sites list, use the `+` to enter the new site creation flow
- procede to step 4, domain suggestions
- Ensure you see suggestions based on the Site Title entered
- Ensure searching for new domains works

Needs review: @ScoutHarris 

_Note:_ The API will be getting changed to support `only_wordpressdotcom` parameter to limit suggestions to only `wpcom` domains. Until that lands, all domains will be suggested.
